src/gmail.d.ts

```diff
@@ -870,7 +870,7 @@ interface GmailObserve {
       trigger the related event and fire off any relevant bound callbacks
       This function should return true if a dom observer is found for the specified action
      */
-    on_dom(action: GmailBindAction, callback: Function): void;
+    on_dom(action: GmailBindAction | T, callback: Function): boolean;
```

Comment:
Coming back to this PR and looking into it... Why do we need to change the return-type?

Does anyone use it for anything? :smile:

Reply:
I just updated the return type declaration here to be consistent with the docs on the function which says it returns a boolean.

Reply:
Fair enough. Let’s make it so!